redaxo/src/addons/mediapool/lib/service_media.php

```diff
@@ -225,4 +231,98 @@ public static function deleteMedia(string $filename): void
             'filename' => $filename,
         ]));
     }
+
+    /**
+     * @param array<array{type: string, value: int}> $searchItems
```

Comment:
sorry, hier hatte ich dir auf skype was falsches gesagt. bei dem äußeren array fehlt der typ für den key.
würde hier aber doch list nehmen (wo der typ festgelegt ist auf durchnummerierte ints):

```suggestion
     * @param list<array{type: string, value: int}> $searchItems
```

Reply:
wobei der typ auch noch nicht komplett so ist, denn value kann scheinbar auch ein array sein statt int.